src/lib/openapi/spec/health-check-schema.ts

```diff
@@ -0,0 +1,16 @@
+import { FromSchema } from 'json-schema-to-ts';
+
+export const healthCheckSchema = {
+    $id: '#/components/schemas/healthCheckSchema',
+    type: 'object',
+    additionalProperties: false,
+    required: ['health'],
+    properties: {
+        health: {
+            enum: ['GOOD', 'BAD'],
```

Comment:
Should this have `type: 'string'` as well?

Reply:
As it is, the resulting type seems correct to me:
![image](https://user-images.githubusercontent.com/14320932/174566373-60117124-8761-419b-8c47-24bf734e8bf8.png)
This is a response-only type and we know the exact options it can have. Do you still think I should change it to `type: 'string'` instead?